src/platform/backends/qemu/CMakeLists.txt

```diff
@@ -27,25 +32,13 @@ add_library(qemu_backend STATIC
   ${CMAKE_SOURCE_DIR}/include/multipass/process/process.h)
 
 target_link_libraries(qemu_backend
-  daemon
   fmt
   logger
-  Qt5::Core
-  Qt5::Network
-  ip_address
-  process
-  shared_linux
-  ssh
+  qemu_img_utils
+  qemu_platform_detail
   utils
-  yaml)
-
-add_custom_target(copy-check-kvm-support ALL DEPENDS
-  ${CMAKE_BINARY_DIR}/bin/check_kvm_support)
-add_custom_command(OUTPUT ${CMAKE_BINARY_DIR}/bin/check_kvm_support
-  COMMAND ${CMAKE_COMMAND} -E copy
-      ${CMAKE_CURRENT_SOURCE_DIR}/check_kvm_support
-      ${CMAKE_BINARY_DIR}/bin/check_kvm_support
-  DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/check_kvm_support)
+  Qt5::Core)
 
-install(PROGRAMS check_kvm_support
-  DESTINATION bin)
+if(LINUX)
+  add_subdirectory(linux)
+endif()
```

Comment:
This could probably also work by way of `add_subdirectory(${MULTIPASS_PLATFORM})` or so.

Reply:
Yep, fixed and much nicer :slightly_smiling_face: